blueocean-github-pipeline/pom.xml

```diff
@@ -75,4 +75,20 @@
 
   </dependencies>
 
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+                <!-- because test GithubOrgFolderPermissionsTest is using some bad static library
+                     it would be better to fix that but maybe next time -->
+              <reuseForks>false</reuseForks>
```

Comment:
Why? This is long since the default in the parent POM: https://github.com/jenkinsci/plugin-pom/pull/453

Reply:
maybe look at the blueocean itself and not only macro changes from a PR and you will see this https://github.com/jenkinsci/blueocean-plugin/blob/feb97f3796f4f616fec59698f2fe536fd506bc0b/pom.xml#L763

which is perfectly right and definitely faster but not for this module and not for this library called `WireMockRule` (probably because it's a tool old version but here it's a patched branch so touching this is not an option)

Reply:
Yeah `reuseForks=false` is set as the default under the assumption that most tests are going to be `JenkinsRule`-based. It does add some overhead if you have tons of short test cases.